### ISSUE TITLE
New feature requirements

### ISSUE THREAD

**Issue by yangz1998** [eclipse-ecal/fineftp-server#10]
Can you add some new features in your free time?

1、Password validation：After the client enters the user password, it encrypts it, and then verifies it with the cipher text of the server. Only after they match each other can they log in to the server.

2、Log record：The client's request message and connection time as well as the client's operation record and operation time are recorded through the log.

Thank you very much !

**Comment by yangz1998** [comment]
Can you give me a reply? 
Waiting...

**Comment by FlorianReimold** [comment]
Hi yangz1998,

1) This is what usually is referred to as FTPS. I guess one would implement this using openssl. As my use-case (-> [eCAL](https://github.com/continental/ecal)) does not require encryption I will not work on this in the foreseeable future. It's still a nice to have feature, so if anybody would like to work on this I am happy to accept contributions 😉

2) Not sure if a library like this should write log-files. I actually think that it shouldn't. It would still be nice to have some kind of possibility to provide a log-file-writing-method from the outside. Again, as I currently don't need it, I will not start working on this at the moment.

**Comment by yangz1998** [comment]
hi FlorianReimold,
I see. Thank you very much for your guidance！

**Comment by aimybbe** [comment]
Do you have plan  to add  a  cmake build system for arm/aarch ?
